apps/api/src/users/users.service.ts

```diff
@@ -46,12 +46,14 @@ export class UsersService {
   }
 
   async userExists(user: AddUserData): Promise<boolean> {
-    return (
-      (await this.usersRepository.findOneBy([
-        { name: user.name },
-        { email: user.email },
-      ])) !== null
-    );
+    return await this.usersRepository?.createQueryBuilder().
```

Comment:
Can the repository be undefined?

Reply:
Oupsy daisy, I sure hope no since it is coming from dependency injection, my bad